ecosystem/sep-0030.md

```diff
@@ -144,8 +144,14 @@ either. The two issuers are:
 - `SEP-10`: A [SEP-10] Web Auth server for proving high threshold control of an
 account. The [SEP-10] server should be configured to only issue [JWT]s if signers
 prove they meet the high threshold on the account.
-- `External`: Defined by the implementer. Proves control of a phone number,
-email address, or other identity.
+- `External`: Defined by the implementer. Proves possession of an identity that
+  is external to the Stellar network such as a phone number, email address, or
+  any other electronic or physical identity. Any forms of identity may be
+  supported. Authentication may be immediate and synchronous, as is the case with
+  electronic identities such as phone, email, and many OpenID Connect
+  providers. Authentication may be delayed and asynchronous, as might be the
```

Comment:
❓ Do we want to give any sort of recommendation here? Immediate and synchronous are probably better from the user experience perspective, albeit not applicable to all authentication cases.

Reply:
>Do we want to give any sort of recommendation here?

I don't think so, mostly because better user experience is subjective depending on the use case. In the right context slower authentication can actually create a better user experience. Also the SEP should be mostly unopinionated about what methods an implementer picks. Wallets/implementers need to make their own business decisions about that.

Reply:
Yep, good point. Agreed.

Reply:
That's what I was thinking, but I'm not hard set on that if you think that level of meta is worth it? I was planning to send a message to stellar-dev asking for suggestions for what other formats to include.